src/ports/postgres/modules/deep_learning/test/madlib_keras.sql_in

```diff
@@ -38,7 +38,7 @@ copy cifar_10_sample_val from stdin delimiter '|';
 -- update cifar_10_sample_val SET independent_var = array_scalar_mult(independent_var::real[], (1/255.0)::real);
 
 -- Prepare the minibatched data manually instead of calling
--- minibatch_preprocessor_dl since it internally calls array_scalar_mult.
+-- training_preprocessor_dl since it internally calls array_scalar_mult.
```

Comment:
Did we mention above that instead of calling any preprocessor, we manually create the table in this dev check?